src/components/Dissolution/CompleteResolution.vue

```diff
@@ -69,37 +115,65 @@
                     cols="1"
                     class="pt-1 pb-1"
                   >
-                    ({{convertIntToChar(index)}})
+                    ({{ convertIntToChar(index) }})
                   </v-col>
-                  <v-col cols="11" class="ml-n11 pt-1 pb-1">
-                    <span>{{bulletItem}}</span>
+                  <v-col
+                    cols="11"
+                    class="ml-n11 pt-1 pb-1"
+                  >
+                    <span>{{ bulletItem }}</span>
                   </v-col>
                 </v-row>
               </li>
             </ul>
           </span>
-          <u class="help-btn" @click="helpToggle = !helpToggle"><small>Hide Help</small></u>
+          <u
+            class="help-btn"
+            @click="helpToggle = !helpToggle"
+          ><small>Hide Help</small></u>
         </section>
       </v-expand-transition>
     </div>
 
     <!-- Special Resolution Form -->
-    <section id="sample-resolution-section" v-if="isTypeCoop" class="mt-10">
+    <section
+      v-if="isTypeCoop"
+      id="sample-resolution-section"
+      class="mt-10"
+    >
       <header id="sample-resolution-header">
-        <h2>{{getCreateResolutionResource.sampleFormSection.header}}</h2>
+        <h2>{{ getCreateResolutionResource.sampleFormSection.header }}</h2>
       </header>
 
-      <p class="section-description mt-2" v-html="getCreateResolutionResource.sampleFormSection.text"></p>
+      <p
+        class="section-description mt-2"
+        v-html="getCreateResolutionResource.sampleFormSection.text"
+      />
 
       <div class="mt-4">
-        <v-card flat class="py-8 px-6">
+        <v-card
+          flat
+          class="py-8 px-6"
+        >
           <div class="d-flex flex-column flex-sm-row justify-center align-center">
-            <img :src="previewImageSource()" class="preview-image" />
+            <img
+              :src="previewImageSource()"
+              class="preview-image"
+            >
             <div class="px-8" />
             <div class="download-link-container py-5">
-              <v-icon color="primary" class="mt-n1">mdi-file-pdf-outline</v-icon>
-              <a :href="documentURL" download class="ml-1">
-                {{getCreateResolutionResource.sampleFormSection.downloadDocLabel}}
+              <v-icon
+                color="primary"
+                class="mt-n1"
+              >
+                mdi-file-pdf-outline
+              </v-icon>
+              <a
+                :href="documentURL"
```

Comment:
Did SonarCloud complain about this?

Reply:
Looks like SonarCloud isn't running for this UI. @pwei1018 ?

Reply:
Patrick, I think SonarCloud isn't running for Edit UI, either. Can you please enable for both UIs, or guide me through it? Thanks.

Reply:
This conversation doesn't block this PR. Patrick and I will look into this later when we both have a moment.

Karim, resolve?

Reply:
Agreed. Resolving.